plugin/backend.go

```diff
@@ -45,13 +37,13 @@ func Backend() *GcpAuthBackend {
 		oauthScopes: []string{
 			iam.CloudPlatformScope,
 			compute.ComputeReadonlyScope,
+			cloudresourcemanager.CloudPlatformScope,
```

Comment:
Let's make sure we document this change in scope in the website docs and changelog

Reply:
I'm also not really sure we should do this at all - iam.CloudPlatformScope and cloudresourcemanager.CloudPlatformScope are both `https://www.googleapis.com/auth/cloud-platform` which is everything? I don't really know if any of the GCP go libraries have a standard "cloud platform scope" constant that I can just use.

Reply:
`cloud-platform` is the everything scope

Reply:
I usually just make my own constant: https://github.com/hashicorp/vault-plugin-secrets-gcpkms/blob/a7d4151e40a5c1649d41440cadd1fe654df7e61c/config.go#L11